translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -6008,6 +6008,66 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
     return mapValue(BC, Result);
   }
 
+  case OpIAddCarry: {
+    SPIRVBinary *BC = static_cast<SPIRVBinary *>(BV);
+    Value *Val0 = transValue(BC->getOperand(0), F, BB);
+    Value *Val1 = transValue(BC->getOperand(1), F, BB);
+    Value *Add = getBuilder()->CreateIntrinsic(Intrinsic::uadd_with_overflow,
+                                             Val0->getType(), {Val0, Val1});
```

Comment:
{Val0, Val1} ->  { Val0, Val1 }

Reply:
That's wrong for llvm style. Running clang-format on this code leaves no whitespace after the { or before the }.

Reply:
I think we ought to make our own version with some modifications of clang-format. In driver development, place spaces after { or before } is to make the array data clear. Also, there are other places that I'm unhappy with default clang-format: 80-character line limitation, 2 spaces indentation, acronym without camelcase (leading to this ABCDEfgh, not AbcdEfgh)